app/Lib/enum.php

```diff
@@ -499,6 +499,7 @@ class OrgIdentityStatusEnum
 class PeoplePickerModeEnum
 {
   const Sponsor   = 'S';
+  const CoPerson  = 'CP';
```

Comment:
Is this the right name? People Pickers are always picking CO People, the mode is intended to convey the context. So it should probably be something like `GroupMember`, or if there's nothing specific to groups then maybe `All` or `Active`.

Reply:
Good point - I think this mode can be used anywhere to grab a person; perhaps "All" is correct - I don't believe we're limiting the Group Member picker to Active only.

Reply:
Went with `const All       = 'AL';` for now

Reply:
Update committed as of ee3fa84